### PR TITLE
[15.0] [MIG] hr_operating_unit

### PR DESCRIPTION
**Pull request by Rajan-SCS**

### CODE REVIEW

**Comment by nikul-serpentcs** [comment]
@AaronHForgeFlow can you please take a look?

**Review by gurneyalex — APPROVED** [review]

File: hr_operating_unit/README.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+Add the operating unit to Employee
```
Comment (AaronHForgeFlow) [inline hr_operating_unit/README.rst:43]:
```suggestion
Add the operating unit to Employees
```

**Review by AaronHForgeFlow — COMMENTED** [review]
I am not a user of this module but I did a code review. Some nitpickings:
* The file hr_operating_unit.py should be renamed to operating_unit.py operating unit is the name of the module.
* There is a typo in the commit message, it has to be "Migrated"

**Review by SkiBY — APPROVED** [review]

**Review by ps-tubtim — APPROVED** [review]

**Comment by AaronHForgeFlow** [comment]
pre-commit is failing @Rajan-SCS  will you fix it?

**Comment by AaronHForgeFlow** [comment]
Superseded in https://github.com/OCA/operating-unit/pull/548/

**Pull request closed without merging**